src/main/resources/l10n/JabRef_en.properties

```
@@ -2635,6 +2626,31 @@ Estimated\ time\ left\:\ more\ than\ 2\ minutes.=Estimated time left: more than
 You\ find\ information\ about\ the\ privacy\ policy\ here.=You find information about the privacy policy here.
 The\ size\ of\ the\ embedding\ model\ could\ be\ smaller\ than\ written\ in\ the\ list.=The size of the embedding model could be smaller than written in the list.
 Embedding\ model\ has\ to\ be\ provided=Embedding model has to be provided
+After\ the\ file\ will\ be\ ingested,\ you\ will\ be\ able\ to\ chat\ with\ it=After the file will be ingested, you will be able to chat with it
+Could\ not\ find\ path\ for\ a\ linked\ file\ '%0'\ while\ generating\ embeddings=Could not find path for a linked file '%0' while generating embeddings
```

I think, these should be complete sentences.

```suggestion
After\ the\ file\ will\ be\ ingested,\ you\ will\ be\ able\ to\ chat\ with\ it.=After the file will be ingested, you will be able to chat with it.
Could\ not\ find\ path\ for\ a\ linked\ file\ '%0'\ while\ generating\ embeddings.=Could not find path for a linked file '%0' while generating embeddings.
```

Don't know about the others.